### DISCUSSION TITLE
Looking for a small bit of help converting outputs

### DISCUSSION THREAD

**Discussion by Crsarmv7l** [bengtmartensson/IrScrutinizer#461] [Q&A] [answered]
Long time user and appreciate the ability to convert to a number of formats. I ran into one the other day that I am not quite sure how to convert (to say RAW).

A device reads the remote and outputs the following: Samsung32 A:0x07 C:0x02

I would like to translate that to RAW

Appreciate any guidance you can provide.

**Comment by bengtmartensson** [comment]
I am afraid that the question does not have a unique and simple answer.  I do not know. You have to ask the author. 

If you give a some context, it may be possible to guess...

**Reply by Crsarmv7l** [reply]
I was able to get some more info from a debug menu:

protocol: Samsung32
address: 07 00 00 00
command: 02 00 00 00

**Reply by bengtmartensson** [reply]
Are you using IRremote (that is producing "Samsung32")  by any chance?

**Reply by Crsarmv7l** [reply]
It doesn't look like it. But I could be wrong. The exact source code for decoding with the device is here:

https://github.com/flipperdevices/flipperzero-firmware/blob/4d6b170769a33711e62deaf1c2a6b3bdda37e665/lib/infrared/encoder_decoder/samsung/infrared_decoder_samsung.c

**Reply by bengtmartensson** [reply]
What they call Samsung 32 is almost certainly what is called NECx-f16 in IrScrutinizer. How to map the parameters is not clear to me; possibly you just make some experiments.

**Comment by bengtmartensson** [comment, marked as answer]
Is this still open?
